### PR TITLE
Add compliant and noncompliant examples of module-injection@v1.0, sen…

### PR DESCRIPTION
**Pull request by prajaki**
*Description of changes:*
* Rules Added: python\module-injection@v1.0, python\sensitive-information-leak@v1.0
* Added new methods for compliant and non compliant examples

### CODE REVIEW

File: src/python/detectors/module_injection/module_injection.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=module-injection@v1.0 defects=1}
+def module_injection_noncompliant():
+    import yaml
+    var = input()
+    # Noncompliant: Load_all without processing is used.
```
Comment (awsgaucho) [inline src/python/detectors/module_injection/module_injection.py:8]: use `load_all` in lowercase

File: src/python/detectors/module_injection/module_injection.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    yaml.load_all(var)
+# {/fact}
+
+
+# {fact rule=module-injection@v1.0 defects=0}
+def module_injection_compliant():
+    import socket
+    var = input()
+    var = internalProcessing(var)
+    # Compliant: Load_all after processing is used.
```
Comment (awsgaucho) [inline src/python/detectors/module_injection/module_injection.py:18]: use `load_all` in lowercase

File: src/python/detectors/sensitive_information_leak/sensitive_information_leak.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    s.bind(('', 0))
+# {/fact}
+
+
+# {fact rule=sensitive-information-leak@v1.0 defects=0}
+def sensitive_information_leak_compliant():
+    import socket
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    # Compliant: Non-empty IP Address is passed when binding to a socket.
+    s.bind(('192.168.1.1', 0))
```
Comment (zjiamzn) [inline src/python/detectors/sensitive_information_leak/sensitive_information_leak.py:18]: Are you sure this is the correct code example?

File: src/python/detectors/module_injection/module_injection.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=module-injection@v1.0 defects=1}
+def module_injection_noncompliant():
+    import yaml
+    input_data = input()
+    # Noncompliant: load_all without processing is used.
```
Comment (san2488) [inline src/python/detectors/module_injection/module_injection.py:8]: Reading this without context, I can't really tell what we mean by `processing` here. I'm also not sure how this describes the `module-injection` issue. Let's discuss offline.

File: src/python/detectors/module_injection/module_injection.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    yaml.load_all(input_data)
+# {/fact}
+
+
+# {fact rule=module-injection@v1.0 defects=0}
+def module_injection_compliant():
+    import socket
```
Comment (san2488) [inline src/python/detectors/module_injection/module_injection.py:15]: should this be `import yaml`?

**Review by grl-wang — APPROVED** [review]

**Pull request merged**